packages/graph/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Rust artifacts
+target/
+
+# Application data
+/deployment/postgres/var/
```

Comment:
I think we want to gitignore `.env.local`. Although it's already checked in/tracked, would be nice if it's still included but not tracked locally

Reply:
Forgot to add this here as well, thank you!

Reply:
The `.env.local` needs to be there for the docker-compose to work (it references `.env.local`). We probably want to add to some readme to `touch .env.local` :)

Reply:
> Although it's already checked in/tracked

I think if you add it to git then gitignoring it doesn't actually work how you want it to... At least it didn't last time I checked (even though I swear I used it like that before). If you add it to the gitignore, it'll still track changes if it's already been check-in.